pkgs/tools/security/pinentry/default.nix

```
@@ -2,7 +2,9 @@
 , libgpgerror, libassuan, qtbase, wrapQtAppsHook
 , ncurses, gtk2, gcr
 , libcap ? null, libsecret ? null
-, enabledFlavors ? [ "curses" "tty" "gtk2" "qt" "emacs" ] ++ lib.optionals stdenv.isLinux [ "gnome3" ]
+, enabledFlavors ? [ "curses" "tty" "gtk2" "emacs" ] 
```

```suggestion
, enabledFlavors ? [ "curses" "tty" "gtk2" "emacs" ]
```
you forgot a trailing whitespace